unstructured/ingest/cli/cmds/weaviate.py

```diff
@@ -0,0 +1,58 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.interfaces import CliMixin, DelimitedString
+from unstructured.ingest.interfaces import BaseConfig
+
+CMD_NAME = "weaviate"
+
+
+@dataclass
+class WeaviateCliWriteConfig(BaseConfig, CliMixin):
+    host_url: str
+    class_name: str
+    auth_keys: t.Optional[t.List[str]] = None
+    additional_keys: t.Optional[t.List[str]] = None
+
+    @staticmethod
+    def get_cli_options() -> t.List[click.Option]:
+        options = [
+            click.Option(
+                ["--host-url"],
+                required=True,
+                help="Weaviate instance url",
+            ),
+            click.Option(
+                ["--class-name"],
+                default=None,
+                type=str,
+                help="Class to ",
+            ),
+            click.Option(
```

Comment:
can we not also support passing these values in directly? When this is being used via python rather than a CLI, needing to use env vars can be a burder.